src/Http/Controllers/VoyagerRoleController.php

```diff
@@ -57,6 +160,9 @@ public function store(Request $request)
         }
 
         if (!$request->ajax()) {
+            if ($request->input('order') < Auth::user()->role->order) {
+                return back()->with(['errors' => collect(['Order must be grater than your order - your order is '.Auth::user()->role->order])]);
```

Comment:
Couple things here....

1. You misspelled `greater`
2. Error messages need to be translated, and shouldn't be hard-coded, especially not in the controller.
3. I don't really like referring to it as "your order".  Makes it seem like you're purchasing something.

Reply:
Thanks.
I solved this notes and change the term from "order" to "rank"